gguf-py/gguf/constants.py

```diff
@@ -693,6 +695,23 @@ class MODEL_TENSOR(IntEnum):
         MODEL_TENSOR.FFN_DOWN_EXP,
         MODEL_TENSOR.FFN_UP_EXP,
     ],
+    MODEL_ARCH.OLMO: [
```

Comment:
I don't understand these mapping with the existing tensors:

https://huggingface.co/allenai/OLMo-7B-hf?show_tensors=model.safetensors.index.json

Reply:
yeah, I've removed the unnecessary ones